lib/membrane_rtmp_plugin/rtmp/source/bin.ex

```diff
@@ -1,9 +1,10 @@
 defmodule Membrane.RTMP.SourceBin do
   @moduledoc """
-  Bin responsible for spawning new RTMP server.
+  Bin responsible for demuxing and parsing a video stream.
```

Comment:
I would add `parsing an RTMP stream`. Also it doesn't have to be video only, it can be both audio video or audio alone.

Reply:
done